src/storyMap/components/StoryMapForm/EditableMedia.js

```diff
@@ -441,6 +441,79 @@ const EditableAudio = props => {
   );
 };
 
+const EditableVideo = props => {
+  const { t } = useTranslation();
+  const [id, setId] = useState(0);
+  const { getMediaFile } = useStoryMapConfigContext();
+  const { video, onUpdate, onDelete, processing } = props;
+
+  useEffect(() => {
+    setId(uuidv4());
+  }, [video]);
+
+  const videoSrc = useMemo(() => {
+    if (video.signedUrl) {
+      return video.signedUrl;
+    }
+    if (video.contentId) {
+      return getMediaFile(video.contentId)
+    }
+    return null;
+  }, [video, getMediaFile]);
+
+  return (
+    <Stack>
+      <video key={id} style={{ width: '100%' }} controls>
```

Comment:
Issue: It seems you need to add the same omit we have for audio: {/* eslint-disable-next-line jsx-a11y/media-has-caption */}